modules/effects/arrow-effects/src/main/kotlin/arrow/effects/MonadSuspendCancellableContinuations.kt

```diff
@@ -30,7 +30,7 @@ open class MonadSuspendCancellableContinuation<F, A>(SC: MonadSuspend<F>, overri
             invoke(f).bind()
 
     suspend fun <B> bindDeferIn(context: CoroutineContext, f: () -> B): B =
-            suspend { bindingCatch { yields(bindIn(context, f)) } }.bind()
+            suspend { bindingCatch { bindIn(context, f) } }.bind()
```

Comment:
Why this change?

Reply:
I clean up some files to prevent the warning regarding `yields` not needed in monad comprehension, this is one of those. Do you agree on the `yields` use there is not needed?

Reply:
Oh I didn't see the bindIn was already there. Yes.